podman/domain/manifests.py

```diff
@@ -0,0 +1,247 @@
+"""Model and Manager for Manifest resources."""
+import urllib.parse
+from typing import List, Optional, Union
+
+import requests
+
+from podman import api
+from podman.api import APIClient
+from podman.domain.images import Image
+from podman.domain.manager import Manager, PodmanResource
+from podman.errors.exceptions import APIError, ImageNotFound, NotFound
+
+
+class Manifest(PodmanResource):
+    """Details and configuration for a manifest managed by the Podman service."""
+
+    @property
+    def id(self) -> str:
+        """Returns the identifier of the manifest."""
+        try:
+            return self.attrs["manifests"][0]["digest"]
+        except (KeyError, TypeError):
+            return ""
```

Comment:
Possibly my ignorance...Should an empty-string `ID` be allowed?  In my (limited) mind, you should allow an exception to propagate if ever referencing an item w/o an ID.

Reply:
@cevich I'm trying to create a jira card to fix this `feature` of the manifest endpoints.